### PR TITLE
[Discipline Priest] Implementation of Conduit Shining Radiance

### PR DESCRIPTION
**Pull request by karlpralow**

### CODE REVIEW

File: src/parser/priest/discipline/modules/shadowlands/conduits/ShiningRadiance.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    if (!this.conduitRank) {
+      this.active = false;
+      return;
+    }
+
+    this.addEventListener(Events.heal.by(SELECTED_PLAYER).spell(SPELLS.POWER_WORD_RADIANCE), this.onRadianceHeal);
+  }
+
+  onRadianceHeal(event: HealEvent) {
+    this.bonusRadianceHealing += calculateEffectiveHealing(event,SHINING_RADIANCE_INCREASE[this.conduitRank]);
```
Comment (abelito75) [inline src/parser/priest/discipline/modules/shadowlands/conduits/ShiningRadiance.tsx]: should probably just make `SHINING_RADIANCE_INCREASE[this.conduitRank]` a constant itself seems extra to keep accessing it from an array when its just going to grab the same number every time

**Review by abelito75 — CHANGES_REQUESTED** [review]
Could you send a screen shot of the how the graphic looks as well?

File: src/parser/priest/discipline/modules/shadowlands/conduits/ShiningRadiance.tsx (hunk 1 above)
Comment (karlpralow) [reply src/parser/priest/discipline/modules/shadowlands/conduits/ShiningRadiance.tsx]: image

**Review by abelito75 — APPROVED** [review]

File: src/parser/priest/discipline/modules/shadowlands/conduits/ShiningRadiance.tsx (hunk 1 above)
Comment (abelito75) [reply src/parser/priest/discipline/modules/shadowlands/conduits/ShiningRadiance.tsx]: beautiful

**Pull request merged**